apps/proxy/src/app/ngrx/store/reducer/root.reducer.ts

```diff
@@ -109,5 +117,27 @@ const _rootReducer = createReducer(
             clientsInProcess: false,
             swtichClientSuccess: false,
         };
+    }),
+
+    on(rootActions.getAllProject, (state) => {
+        return {
+            ...state,
+            projectInProcess: true,
+        };
+    }),
+    on(rootActions.getAllProjectSuccess, (state, { response }) => {
+        return {
+            ...state,
+            projects: Boolean(Object.keys(response.data ?? {})?.length),
```

Comment:
No need to save this in store it can be done in selector
and give its selector proper name